governance/votes/Executive vote - May 30, 2024.md

```diff
@@ -0,0 +1,127 @@
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Smart Burn Engine (SBE) Parameter Updates
+
+- **Authorization:** [Forum post](https://forum.makerdao.com/t/smart-burn-engine-vow-hump-surplus-buffer-upper-limit-reconfiguration-update-7/24348/2)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/smart-burn-engine-vow-hump-surplus-buffer-upper-limit-reconfiguration-update-7/24348)
+
+If this executive proposal passes, the following SBE parameter will be changed:
+
+- The [`vow.hump` (SBE Surplus Buffer Upper Limit)](https://mips.makerdao.com/mips/details/MIP104#9-1-smart-burn-engine-parameters) parameter will be increased by 5 million DAI from 50 million DAI to **55 million DAI**.
+
+### Approve RWA Foundation (RWAF) DAO Resolution
+
+- **Authorization:** [Forum post](https://forum.makerdao.com/t/dao-resolution-banking-setup-for-rwa-foundation/24362/2)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/dao-resolution-banking-setup-for-rwa-foundation/24362)
+
+If this executive proposal passes, the following DAO Resolution pertaining to RWA Foundation (RWAF) will be approved:
+
+- RWAF DAO Resolution with IPFS hash [Qmb8vLDH6wT4Y2axnJX1JSKVKHTG3jzX3U3novw886H8UR](https://ipfs.io/ipfs/Qmb8vLDH6wT4Y2axnJX1JSKVKHTG3jzX3U3novw886H8UR).
+
+### Approve Harbor Trade Credit (RWA004-A) DAO Resolution
+
+- **Authorization:** [Forum post](https://forum.makerdao.com/t/harbor-trade-credit-workout-process/24367/4)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/harbor-trade-credit-workout-process/24367)
+
+If this executive proposal passes, the following DAO Resolution pertaining to [Harbor Trade Credit (RWA004-A)](https://makerburn.com/#/collateral/RWA004-A) will be approved:
+
+- RWA004-A DAO Resolution with IPFS hash [QmUiYTRy4BkV681tfFe3Ksj6gdLEq27w34MqLw5LvRaBoD](https://ipfs.io/ipfs/QmUiYTRy4BkV681tfFe3Ksj6gdLEq27w34MqLw5LvRaBoD).
+
+### Trigger Spark Proxy Spell
+
+- **Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327/3), [Poll](TBD)
+- **Proposal**: [Forum posts](https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327)
+
+If this executive proposal passes, a Spark proxy spell at [TBD](https://etherscan.io/address/TBD) will be triggered. This proxy spell will take the following actions:
+
+#### [Mainnet+Gnosis] Turn off silo borrowing for all stablecoins
+
+- Turn off [siloed borrowing](https://docs.sparkprotocol.io/developers/sparklend/features/siloed-borrowing) for Circle USD (USDC) and Tether USD (USDT) on both Ethereum and Gnosis.
+- Turn off siloed borrowing for Monerium EURO e-money (EURe) on Gnosis.
+
+#### [Mainnet+Gnosis] IRM (Interest Rate Model) Update
```

Comment:
As per comment on the poll PR, https://github.com/makerdao/community/pull/2652#discussion_r1616028463 consider adding the deltas for changing parameters